src/content/docs/fr/develop-and-build.mdx

```diff
@@ -0,0 +1,146 @@
+---
+title: Développer et construire
+description: 'Comment commencer à travailler sur un nouveau projet.'
+i18nReady: true
+---
+import { Tabs, TabItem, FileTree, CardGrid, LinkCard, Steps } from '@astrojs/starlight/components';
+import ReadMore from '~/components/ReadMore.astro';
+
+Une fois que vous avez un projet Astro, vous êtes prêt à construire avec Astro ! 🚀
+
+## Modifier votre projet
+
+Pour apporter des modifications à votre projet, ouvrez votre dossier de projet dans votre éditeur de code. Travailler en mode développement avec le serveur de développement en marche vous permet de voir les mises à jour de votre site au fur et à mesure que vous modifiez le code.
+
+Vous pouvez également [personnaliser les aspects de votre environnement de développement](#configurer-votre-environnement-de-développement) telles que la configuration de TypeScript ou l'installation des extensions officielles de l'éditeur Astro.
+
+### Démarrer le serveur de développement Astro
+
+Astro est livré avec un serveur de développement intégré qui contient tout ce dont vous avez besoin pour le développement de projets. La commande `astro dev` CLI démarrera le serveur de développement local afin que vous puissiez voir votre nouveau site en action pour la première fois.
```

Comment:
nit: I think it's more correct to put `CLI` before... at least it's easier to read:
```suggestion
Astro est livré avec un serveur de développement intégré qui contient tout ce dont vous avez besoin pour le développement de projets. La commande CLI `astro dev` démarrera le serveur de développement local afin que vous puissiez voir votre nouveau site en action pour la première fois.
```